src/NHibernate.Test/Logging/LoggerPerformanceTest.cs

```diff
@@ -120,13 +120,15 @@ public INHibernateLogger LoggerFor(System.Type type)
 			}
 		}
 
-		private class MockLogger2 : INHibernateLogger
+#pragma warning disable 618
+		private class MockLogger2 : NHibernateLoggerBase, INHibernateLogger
+#pragma warning restore 618
 		{
-			private bool IsErrorEnabled { get; } = true;
-			private bool IsFatalEnabled { get; } = true;
-			private bool IsDebugEnabled { get; } = false;
-			private bool IsInfoEnabled { get; } = false;
-			private bool IsWarnEnabled { get; } = true;
+			private new bool IsErrorEnabled { get; } = true;
```

Comment:
That hides the base class member, but we do not care. The base class member still serves as the interface member.